docs/appending_events.md

```diff
@@ -0,0 +1,152 @@
+# Appending Events
+
+## Append your first event
+
+The simplest way to append an event is to create an event object, a stream object and call client's `#append_to_stream` method.
+
+```ruby
+require 'securerandom'
+
+class SomethingHappened < PgEventstore::Event
+end
+
+event = SomethingHappened.new(data: { user_id: SecureRandom.uuid, title: "Something happened" })
+stream = PgEventstore::Stream.new(context: 'MyAwesomeContext', stream_name: 'SomeStream', stream_id: '1')
```

Comment:
Consistent with what? `stream_id` is a string - there can be anything

Reply:
Yes, but we should probably show the best practice of using uuids everywhere. This might encourage some people to use ints instead of uuids. So i agree with Arek.